lib/cartopy/tests/crs/test_nearside_perspective.py

```diff
@@ -23,28 +23,48 @@
 
 from numpy.testing import assert_almost_equal
 
-from cartopy.tests.crs.test_geostationary import (GeostationaryTestsMixin,
-                                                  check_proj4_params)
+from cartopy.tests.crs.test_geostationary import check_proj4_params
 
-from cartopy.crs import NearsidePerspective
+import cartopy.crs as ccrs
 
 
-class TestEquatorialDefault(GeostationaryTestsMixin, object):
-    # Check that it behaves just like Geostationary, in the absence of a
-    # central_latitude parameter.
-    test_class = NearsidePerspective
-    expected_proj_name = 'nsper'
+class TestNearsidePerspective(object):
+    def test_default(self):
```

Comment:
Since there is no longer any subclassing, might as well write these at the top-level.

Reply:
Done.

Reply:
OH, I understand better now.

Reply:
Thought you meant the weird subclassing still going on for Geostationary. Apparently reading comprehension is not a strong suit for me.

Reply:
Now actually fixed.